xbmc/music/tags/TagLoaderTagLib.cpp

```diff
@@ -561,7 +561,6 @@ bool CTagLoaderTagLib::ParseTag(Ogg::XiphComment *xiph, EmbeddedArt *art, CMusic
   if (!xiph)
     return false;
 
-  FLAC::Picture pictures[3];
```

Comment:
do we still want to support taglib versions less than 1.11?
Then I need to ifedf this variable.